src/test/java/org/folio/moduserstest/RestVerticleIT.java

```diff
@@ -2198,4 +2189,132 @@ private static String makeFakeJWT(String username, String id, String tenant) {
       Base64.getEncoder().encodeToString((header.encode() + payload.encode())
         .getBytes(StandardCharsets.UTF_8)));
   }
+
+  @Test
+  public void test4CustomFields(TestContext context) {
+
+    Async async = context.async();
+    postCustomField(context)
+      .compose(v -> putCustomField(context))
+      .compose(v -> queryCustomField(context)).compose(this::assertCustomFieldValues)
+      .compose(v -> deleteCustomField(context))
+      .setHandler(res -> {
+        if (res.succeeded()) {
+          async.complete();
+        } else {
+          res.cause().printStackTrace();
+          context.fail(res.cause());
+        }
+    });
+  }
+
+  private Future<Void> assertCustomFieldValues(JsonObject result) {
+    Future<Void> future = Future.succeededFuture();
+    int totalRecords = result.getInteger("totalRecords");
+    if (totalRecords != 1) {
+      future.fail("Expected 1 record, got " + totalRecords);
+    }
+    JsonArray customFields = result.getJsonArray("customFields");
+    JsonObject customField = customFields.getJsonObject(0);
+    assertEquals(customField.getString("entityType"), "user");
```

Comment:
The assert parameters are

    assertEquals(expected, actual);

or

    assertThat(actual, is(expected));

Therefore the correct parameter order is

    assertThat(customField.getString("entityType"), is("user"));

or

    assertEquals("user", customField.getString("entityType"));

Reply:
changed, thanks @julianladisch